lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -863,6 +863,9 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
 
     # Perform the actual upload
     upload_csv_to_report_store(rows, 'grade_report', course_id, start_date)
+    TASK_LOG.info(u'Grade reported generated for course %s',
```

Comment:
We don't have any log before that let's us know about a grade report being generated. We can just search logs by course_id and we will be able to track all grade reports generated for that course.